cmd/lotus-chainwatch/scheduler/refresh_top_miners_by_base_reward.go

```diff
@@ -8,6 +8,56 @@ import (
 	"golang.org/x/xerrors"
 )
 
+func setupTopMinerByBaseRewardSchema(ctx context.Context, db *sql.DB) error {
+	select {
+	case <-ctx.Done():
+		return nil
+	default:
+	}
+
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+	if _, err := tx.Exec(`
+		create materialized view if not exists top_miners_by_base_reward as
+			with total_rewards_by_miner as (
+				select
+					b.miner,
+					sum(bbr.base_block_reward * b.win_count) as total_reward
```

Comment:
Should we also be dividing here?: https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/reward/reward_actor.go#L74

Reply:
Never mind, that happens in `storeBaseBlockReward`

Reply:
Division happens in big.Int land at https://github.com/filecoin-project/lotus/blob/c6e0a69bbc92c19d6825444ba1f8ab5c1040acf0/cmd/lotus-chainwatch/processor/reward.go#L190